### ISSUE TITLE
docs: eslint error tip does not display

### ISSUE THREAD

**Issue by zzxming** [eslint-stylistic/eslint-stylistic#746]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

When the error position is the last char of the line, the error tip trigger element is empty. So we can't use mouse hover the element and trigger the error tip.

I find a repo at current docs page: https://eslint.style/rules/js/semi#always

1. Press F12
2. Select element ` 1` at the `class Foo { bar = 1 }` in incorrect box

You can see a empty `div` after char ` 1` in elements inspect
```html
<div class="v-popper v-popper--theme-twoslash v-popper--theme-dropdown twoslash-error twoslash-error-hover"><span><span></span></span></div>
```

If we edit this element add some word. and the error message will display.

I don't know what this bug is about and how long it's been around. and not just `semi`. Other rules docs also have this bug (for I know [ts/member-delimiter-style](https://eslint.style/rules/ts/member-delimiter-style) also have this bug. But current docs page can't be  reproduced, because other bug).

I guess maybe is `@shikijs/twoslash` problem? I don't know much about Shiki. So I may can't handle this problem.

### Reproduction

https://eslint.style/rules/js/semi#always

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests
